stdlib/public/core/Sequence.swift

```diff
@@ -548,6 +548,10 @@ public protocol Sequence {
   __consuming func _copyContents(
     initializing ptr: UnsafeMutableBufferPointer<Element>
   ) -> (Iterator,UnsafeMutableBufferPointer<Element>.Index)
+
+  func withUnsafeBytesIfSupported<T>(
```

Comment:
@airspeedswift Again this is our presumption of spelling for the `Sequence` methods